dist/toolwindow.js

```diff
@@ -900,6 +929,11 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
         if (!evt || !evt.target) {
           return;
         }
+		this._log('_onMouseDown: '+(this._isDrag?' Drag':'')+(this._isResize?' resize=':'')+this._resizeMode+(this._isButton?' Button':''));		
+
+		if (evt.target.closest(".dialog>.content") && this._resizeMode === '') { //## Allow normal events on content element
```

Comment:
suggest: for robustness (handling erroneous resizeMode like `undefined` or `null`), performance (quicker to test a string vs doing a dom query), and because empty string is falsey:
```
if (!this._resizeMode && evt.target.closest(".dialog>.content")) {
	return;
}
```

Reply:
done